src/ansys/mapdl/core/pool.py

```diff
@@ -286,8 +288,11 @@ def __init__(
                 if not os.path.isdir(self._root_dir):
                     os.makedirs(self._root_dir)
         else:
-            if isinstance(port, int) or len(port) == 1:
-                ports = [port + i for i in range(n_instances)]
+            if isinstance(port, int) or (isinstance(port, list) and len(port) == 1):
+                if len(ip) > 1:
```

Comment:
```suggestion
                if len(ips) > 1:
```

Reply:
I have the feeling it should be the variable `ips` here as `ip` can be a string.

Reply:
You are right!